component documentation/componentDoc.css

```diff
@@ -0,0 +1,190 @@
+:root {
+    --primary-color: white ;
+    --background-color: rgb(29, 27, 27);
+    --secondary-color: #6970e7;
+    --text-color: #737477;
+    --backgroundTwo-color: rgb(34, 32, 32); 
```

Comment:
you can given better naming conventions, --backgroundTwo-color will not be understood easily by others.. and you have property as background-color in css.. it creates  ambiguity for users.. plz mention proper color names

Reply:
ok, i will change that and make more understandable